packages/dd-trace/test/config.spec.js

```diff
@@ -612,30 +631,6 @@ describe('Config', () => {
     expect(config).to.have.property('env', 'test')
   })
 
-  it('should support the serviceMapping environment variable', () => {
```

Comment:
is this test not valid anymore?

Reply:
It's functionally merged into the test that tests all the environment variables together.